build_paths.h.cmake

```diff
@@ -24,32 +24,4 @@
 ///////////////////////////////////////////////////////////////////////////////
 #endif
 
-///////////////////////////////////////////////////////////////////////////////
-//	Other Configuration
-
-//	set global prefix (normal /usr/local)
-//#define PREFIX "/usr/local"
-#cmakedefine PREFIX "${PREFIX}"
```

Comment:
I actually wanted to include that in a seperate commit. Gonna revert this, although it can be removed later